packages/framework/express/package.json

```diff
@@ -1,5 +1,3 @@
 {
-  "main": "../dist/cjs/servers/express.cjs",
```

Comment:
These are not required as ESM sub exports will always be fetched from the top level package.json